File: examples/sentiment/analyze/node/analyzeAPieceOfText.js
const { SentimentService } = require("m3o/sentiment");

// Analyze and score a piece of text
async function analyzeApieceOfText() {
  let sentimentService = new SentimentService(process.env.MICRO_API_TOKEN);
  let rsp = await sentimentService.analyze({
    text: "this is amazing",
  });
  console.log(rsp);
}

analyzeApieceOfText();

File: examples/go/sentiment/analyze/analyzeAPieceOfText.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/sentiment"
)

// Analyze and score a piece of text
func AnalyzeApieceOfText() {
	sentimentService := sentiment.NewSentimentService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := sentimentService.Analyze(&sentiment.AnalyzeRequest{
		Text: "this is amazing",
	})
	fmt.Println(rsp, err)
}
